test/utils_test.js

```diff
@@ -0,0 +1,68 @@
+import { expect } from "chai";
+
+import {cleanRecord, renderDisplayField} from "../scripts/utils";
+
+describe("cleanRecord", () => {
+  it("should remove id, schema and last_modified from properties", () => {
+    expect(cleanRecord({
+      id: "toto",
+      last_modified: 1234,
+      schema: 5638,
+      foo: "bar",
+      foobar: "foo"
+    })).to.eql({
+      foo: "bar",
+      foobar: "foo"
+    });
+  });
+});
+
+describe("renderDisplayField", () => {
+  let record;
+
+  beforeEach(() => {
+    record = {
+      title: "I am a title",
+      extras: {foo: "bar", foobar: "foo"},
+      "faux.ami": "Yes I am",
+      "supported.strange.nested": {"tree": "foobar", "tree.value": "bar"}
+    };
+  });
+  
+  it("should return the title field as a string", () => {
+    expect(renderDisplayField(record, "title")).to.eql("I am a title");
+  });
+
+  it("should return the extras field as a JSON string", () => {
+    expect(renderDisplayField(record, "extras")).to.eql(
+      JSON.stringify({"foo": "bar", "foobar": "foo"}));
```

Comment:
Nit: here and elsewhere in other tests it would be good to always keep this formalism for consistency and to avoid having lines over 80c:

```js
expect(thing)
  .eql(expectation);
```